app/src/main/res/layout/shoplist_row.xml

```diff
@@ -9,17 +9,17 @@
 
     <CheckBox
         android:id="@+id/item_checkbox"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
         android:layout_height="30dp"
         android:layout_weight="1" />
 
     <ImageButton
         android:id="@+id/delete_item_button"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_weight="0.1"
+        android:layout_weight="8"
         android:adjustViewBounds="true"
-        android:background="@color/common_google_signin_btn_text_dark"
+        android:background="#00FFFFFF"
```

Comment:
If i recall correctly you can use "@android:color/transparent" to have a transparent background instead

Reply:
Done